[BUG] Fix cargo tangle

Doesn't work to install currently.

```bash
error[E0277]: the trait bound `generic::header::Header<Number, Hash>: Encode` is not satisfied
    --> /Users/drew/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sp-runtime-34.0.0/src/generic/header.rs:75:39
     |
75   | impl<Number, Hash> traits::Header for Header<Number, Hash>
     |                                       ^^^^^^^^^^^^^^^^^^^^ the trait `Encode` is not implemented for `generic::header::Header<Number, Hash>`, which is required by `generic::header::Header<Number, Hash>: Codec`
     |
     = note: required for `generic::header::Header<Number, Hash>` to implement `Codec`
note: required by a bound in `traits::Header`
    --> /Users/drew/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sp-runtime-34.0.0/src/traits.rs:1199:24
     |
1198 | pub trait Header:
     |           ------ required by a bound in this trait
1199 |     Clone + Send + Sync + Codec + Eq + MaybeSerialize + Debug + TypeInfo + 'static
     |                           ^^^^^ required by this bound in `Header`
help: consider extending the `where` clause, but there might be an alternative better way to express this requirement
     |
78   |     Hash: HashT, generic::header::Header<Number, Hash>: Encode
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0277]: the trait bound `generic::header::Header<Number, Hash>: Encode` is not satisfied
   --> /Users/drew/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sp-runtime-34.0.0/src/generic/header.rs:148:17
    |
148 |         Hash::hash_of(self)
    |         ------------- ^^^^ the trait `Encode` is not implemented for `generic::header::Header<Number, Hash>`
    |         |
    |         required by a bound introduced by this call
    |
note: required by a bound in `traits::Hash::hash_of`
   --> /Users/drew/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sp-runtime-34.0.0/src/traits.rs:973:16
    |
973 |     fn hash_of<S: Encode>(s: &S) -> Self::Output {
    |                   ^^^^^^ required by this bound in `Hash::hash_of`
help: consider extending the `where` clause, but there might be an alternative better way to express this requirement
    |
143 |     Hash: HashT, generic::header::Header<Number, Hash>: Encode
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -63,7 +63,7 @@ cargo-tangle = { path = "./cli", version = "0.3.1" }
 cargo_metadata = { version = "0.18.1" }
 
 # Tangle-related dependencies
-tangle-subxt = { version = "0.6.0", default-features = false }
+tangle-subxt = { version = "0.7.0", default-features = false }
 subxt-signer = { version = "0.37.0", default-features = false }
 subxt = { version = "0.37.0", default-features = false }
 subxt-core = { version = "0.37.0", default-features = false }

